src/InputWithLabel.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useRef } from 'react';
+
+const InputWithLabel = (props) => {
```

Comment:
For coding style consistency, you could destructure the props similar to how you've done so in AddTodoForm and TodoList. No need to make the change for this PR though. It would look something like this:
```javascript
{
  todoTitle,
  handleTitleChange,
  children,
}
```